spec/integration/query_cache_spec.rb

```diff
@@ -513,4 +513,113 @@
       expect(events.length).to eq(2)
     end
   end
+
+  describe 'in transactions' do
+    min_server_fcv '4.0'
+    require_topology :replica_set
```

Comment:
We have a `require_transaction_support` for this.

Reply:
So convenient! Thanks for letting me know.